usr/share/tb-profile-i2p/start-i2p-browser

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env bash
+#
+# GNU/Linux does not really require something like RelativeLink.c
+# However, we do want to have the same look and feel with similar features.
+#
+# Copyright 2017 The Tor Project.  See LICENSE for licensing information.
```

Comment:
What license is it?

https://github.com/Whonix/tb-starter/blob/master/debian/copyright changes may be required.

Reply:
For now, I've just assumed that it's under the same license as Tor itself and added Tor.txt from the browser bundle as LICENSE.tor. I don't think it's tor-launcher(That's for the browser plugin), and I know it's not torbutton, so I think that's it.